<launch>
  <node name="update_frame_id" pkg="hma_pcl_reconst" type="update_frame_id.py" output="screen" />

  <node pkg="nodelet" type="nodelet" args="manager" name="standalone_nodelet" output="screen"/>

  <!-- Convert to point cloud -->
  <node pkg="nodelet" type="nodelet" name="depth_image_proc" args="standalone depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
   
    <remap from="rgb/image_rect_color" to="/camera/color/image_raw"/>
    <remap from="depth_registered/image_rect" to="/camera/aligned_depth_to_color/image_raw"/>
    <remap from="points" to="/camera/depth_registered/points"/>

  </node>

</launch>
